blueprints/ember-cli-notifications/index.js

```diff
@@ -7,9 +7,11 @@ module.exports = {
   // Use config to determine whether Font Awesome is imported into consuming app
   afterInstall: function(app) {
     var projectConfig = this.project.config(app.env);
-    var config = projectConfig['ember-cli-notifications'];
+    var config = projectConfig['ember-cli-notifications'] ||
+                 app.options['ember-cli-notifications'] ||
```

Comment:
Why separate? Let's use the config object in the application's `environment.js`.